userguide/one_shot_object_detection/README.md

```diff
@@ -44,81 +44,67 @@ model.save('mymodel.model')
 model.export_coreml('MyCustomOneShotDetector.mlmodel')
 ```
 
-Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
-
-
-```python
-augmented_images = \
-	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
-
-augmented_images.explore()
-```
-
-![Explore augmented images](images/augmented_images_explore.jpg)
-
-Some examples of synthetic training data generated:
-
-![Sample augmented images](images/augmented_images_collage.jpg)
-
-*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
 
-
-To visualize the predictions made on the test set:
-
-
-```python
-test_data['image_with_predictions'] = \
-	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
-
-test_data.explore()
-```
+Visualizing predictions made on the test set:
```

Comment:
I don't think this explore on the test set is adding much value. I'd leave remove it.